addOns/commonlib/src/test/java/org/zaproxy/addon/commonlib/DiceMatcherUnitTest.java

```diff
@@ -132,4 +136,21 @@ void shouldGiveCorrectPercentageForDifferentString() {
         // Then
         assertThat(sim, is(equalTo(25)));
     }
+
+    static Stream<Arguments> stringArraySource() {
+        return Stream.of(
+                Arguments.of((Object) new String[] {null, ORIGINAL_STRING}),
+                Arguments.of((Object) new String[] {ORIGINAL_STRING, null}),
+                Arguments.of((Object) new String[] {"ab", ORIGINAL_STRING}),
+                Arguments.of((Object) new String[] {ORIGINAL_STRING, "ab"}));
+    }
+
+    @ParameterizedTest
+    @MethodSource("stringArraySource")
+    void shouldGiveZeroPercentageForNullOrShortString(String[] in) {
```

Comment:
- Should be `String stringA, String stringB` (or with other names).
 - `...OrCompletelyDifferentString` does it matter if it's short?

Reply:
It's an array being passed. Do you mean in the given I should do `String strA = in[a]` then use `strA` for the remainder of the test?

Reply:
The source should return two arguments (instead of an array).

Reply:
Okay I'll try to sort that out :wink:

Reply:
Don't create the arrays, just pass the strings.

Reply:
But how to pass two sets of strings, or generate the wanted (AB, BA, AA) combinations... I'm sure there's a simple solution. I just gotta research a bit.

Reply:
Do `Arguments.of(null, ORIGINAL_STRING}), Arguments.of(...)` (remove the arrays).

Reply:
Thanks, I knew it had to be something simple 😀

Reply:
The comment about the test method name was not addressed (I'm fine with that name if that's really just testing a short string and not a completely different one).

Reply:
I had named it like that specifically, because I intended that test method do be based around the initial few checks (null, short strings, etc)

However, I guess there's no reason that it couldn't include other zero percent options.

Reply:
OK, that sounds good to me, no changes needed.